apps/s_client.c

```diff
@@ -1963,7 +1963,7 @@ int s_client_main(int argc, char **argv)
 
     /*
      * In TLSv1.3 NewSessionTicket messages arrive after the handshake and can
-     * come at any time. Therefore we use a callback to write out the session
+     * come at any time. Therefore, we use a callback to write out the session
```

Comment:
Commas after therefore don't really count as typos - certainly in this case, I would not use a comma here. Possibly in the second. The convention when things can be ambiguous is to leave them as written by the author